server/metrics/metrics.go

```diff
@@ -38,6 +38,12 @@ const (
 	// CRIOImageLayerReuseKey is the key for the CRI-O image layer reuse metrics.
 	CRIOImageLayerReuseKey = "crio_image_layer_reuse"
 
+	// CRIOContainersOOMKey is the key for the CRI-O container out of memory metrics.
+	CRIOContainersOOMKey = "crio_containers_oom"
```

Comment:
Both should have _total suffix.

Reply:
Removed that metric as per https://github.com/cri-o/cri-o/pull/4690#pullrequestreview-623167085